docs/channels/emails.rst

```diff
@@ -225,15 +225,53 @@ Marketers can place the signature into an Email using the ``{signature}`` token.
 Email delivery
 ##############
 
-Mautic delivers Emails using the method defined by the system administrator. If you are the system administrator for your Company, then you need to add the Email protocol for your Mautic instance to use. Mautic integrates with any Email service provider which offers SMTP mail servers as well as several distinct services such as:
+Mautic delivers Emails using the method defined by the system administrator. If you are the system administrator for your Company, then you need to add the Email protocol for your Mautic instance to use. Mautic integrates with any Email service provider which offers SMTP mail servers. As Mautic uses the :xref:`Symfony Mailer` library since v5, it supports its core plugins. Transports for other email services might be found on Github or Packagist.
 
-- :xref:`Mandrill`
-- :xref:`GMail`
-- :xref:`Sendgrid`
-- :xref:`Mailjet`
-- :xref:`Postmark`
-- :xref:`Sendmail`
-- :xref:`Amazon SES`.
+SMTP Transport
+**************
+
+The SMTP transport is the default transport for Mautic. It's configured in the Mautic configuration under the Email Settings tab. The configuration is the same as in the :xref:`Symfony Mailer` documentation.
+
+This is the example DSN mentioned in the :xref:`Symfony Mailer` documentation for SMTP:
+
+.. code-block:: shell
+    
+    smtp://user:pass@smtp.example.com:port
+
+This is how this DSN converted to Mautic's email configuration:
+
+.. image:: images/emails/smtp-dsn.png
+    :width: 400
+    :alt: SMTP API DSN example
+
+.. note::
+  Use the Mautic's global configuration to paste in the DSN information. Especially the API keys and passwords. The values must be URL-encoded and the configuration form will do that for you. If you are pasting DSN directly into the config/local.php file, you must URL-encode the values yourself.
+
+Example API Transport Installation
+**********************************
+
+Let's say you want to use :xref:`Sendgrid` API instead of SMTP to send Emails. You can install the official Symfony Sendgrid Transport by running the following command that is mentioned along others in the :xref:`Symfony Mailer` documentation.
+
+.. code-block:: shell
+    
+    composer require symfony/sendgrid-mailer
```

Comment:
What about people who are not using Composer? How do they use it?

Reply:
I don't think it makes sense to use those without Composer. Non-Composer Mautic installations will have to go for a Mautic plugin that does not have any other dependencies if SMTP is not enough.

Reply:
I think that we need to make this clear up front then, that you need to be using composer? Or will be be supplying such plugins?

Reply:
I added a warning in https://github.com/mautic/user-documentation/pull/174/commits/e1850808dc6e8362c6338ec051280e2dd80c8e50